charm/bundle/bundles.yaml

```diff
@@ -0,0 +1,21 @@
+calico-testing-formation:
+  services:
+    etcd:
+      charm: cs:~kubernetes/trusty/etcd
+      num_units: 1
+    docker:
+      charm: cs:trusty/docker
+      num_units: 2
+      options:
+        latest: true
+        version: 1.6.2
+    calico-docker:
+      charm: local:trusty/calico-docker
+      options:
+        docker-image: calico/node:latest
```

Comment:
Ah - just read the readme below. If this file is just used for testing then latest might be the right docker image to use.